Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -79,16 +79,24 @@ private enum CardinalAxisType
         }
 
         /// <summary>
-        /// This enum is used internally to define how an object's bounds are calculated in order to fit the boundingbox
-        /// to it.
+        /// This enum defines what volume type the bound calculation depends on and its priority
+        /// for it.
+        /// RendererOverCollider: Used Renderers for the bounds calculation and Colliders as a fallback
+        /// ColliderOverRenderer: Used Colliders for the bounds calculation and Renderers as a fallback
+        /// ColliderOnly: Omits Renderers and uses Colliders for the bounds calculation exclusively
+        /// ColliderOnly: Omits Colliders and uses Renderers for the bounds calculation exclusively
         /// </summary>
-        private enum BoundsCalculationMethod
+        public enum BoundsCalculationMethod
         {
-            Collider = 0,
-            Colliders,
-            Renderers,
-            MeshFilters
+            RendererOverCollider = 0,
```

Comment:
Can you move the summary comments to each enum? i.e
/// <summary> blah </summary>
RendererOverCollider = 0,
/// <summary> blah </summary>
ColliderOverRenderer , etc

This way they should auto show up in the doc description. example although blank:
https://microsoft.github.io/MixedRealityToolkit-Unity/api/Microsoft.MixedReality.Toolkit.UI.BoundingBox.WireframeType.html